sr-olthad/sr_olthad/framework/lms.py

```
@@ -0,0 +1,194 @@
+                if chunk.choices and chunk.choices[0].delta.content is not None:
+                    chunk_text = chunk.choices[0].delta.content
+                    stream_handler(chunk_text)
+                    full_response += chunk_text
+            return full_response
+        else:  # No need to stream
+            chat_completion: ChatCompletion = await self.client.chat.completions.create(
+                model=self.model, messages=messages, **kwargs
+            )
+            return chat_completion.choices[0].message.content
+
+from huggingface_hub import AsyncInferenceClient
```

Ruff (0.8.2)

Module level import not at top of file

(E402)